e2e/src/main.rs

```diff
@@ -0,0 +1,561 @@
+use http_utils::{http_request, http_response};
```

Comment:
It is cosmetic, but could you format your input using the following convention:
- one shared `use std::{..};`
- one import per crate
- one import for crate `use crate::{..};`

Reply:
This!

The order of use statements goes in this order:

1. std
2. extern crates
3. workspaces
4. within-workspace use statements

the `mod` statements should be either above, or underneath, all these.